plugins/cliconf/eos.py

```diff
@@ -193,8 +193,9 @@ def edit_config(
         commit=True,
         replace=None,
         comment=None,
-        timer=None,
+        **kwargs,
```

Comment:
my only concern around this change is the fact that the only keyword argument that exists is `timer` - and i think i would rather refactor all the calls to `load_config` rather than introduce `kwargs` to this function.